packages/client/src/client4.ts

```diff
@@ -1666,9 +1666,9 @@ export default class Client4 {
         );
     };
 
-    getAllChannelsMembers = (userId: string) => {
+    getAllChannelsMembers = (userId: string, page = 0, perPage = PER_PAGE_DEFAULT) => {
```

Comment:
Is this going to change the default behaviour of this call since we're now adding a page number and page size to it? We have to be careful about that here since this API is public

Reply:
@hmhealey are you talking about server-side implications or Webapp side only

Reply:
I mean, will it affect other people using this API directly from Client4. It's intended that people building plugins and bots would be using Client4 directly, so I'm trying to figure out if the new default values will result in the behaviour changing for anyone else using this

Reply:
Yeah, Practically no effect as currently if you don’t provide defaults API will return the First 60 results